sdk/pyproject.toml

```diff
@@ -32,6 +34,7 @@ dev-dependencies = [
     "pytest==7.4.4",
     "pytest-cov==4.1.0",
     "pytest-asyncio==0.21.2",
+    "rich>=13.9.4",
```

Comment:
what's this for?

Reply:
I used it on local for pretty printing on the console, I can remove it, its just a dev dep